packages/fiori/src/Wizard.ts

```diff
@@ -251,7 +188,82 @@ const metadata = {
  * @appenddocs sap.ui.webc.fiori.WizardStep
  * @public
  */
+@customElement("ui5-wizard")
+@languageAware
+@fastNavigation
+
 class Wizard extends UI5Element {
+	/**
+	 * Defines the width of the <code>ui5-wizard</code>.
+	 * @private
+	 */
+	@property({ validator: Float })
+	width!: number
+
+	/**
+	 * Defines the threshold to switch between steps upon user scrolling.
+	 * <br><br>
+	 *
+	 * <b>For Example:</b>
+	 * <br>
+	 * (1) To switch to the next step, when half of the step is scrolled out - set <code>step-switch-threshold="0.5"</code>.
+	 * (2) To switch to the next step, when the entire current step is scrolled out - set <code>step-switch-threshold="1"</code>.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Supported values are between 0.5 and 1
+	 * and values out of the range will be normalized to 0.5 and 1 respectively.
+	 * @private
+	 * @type {sap.ui.webc.base.types.Float}
+	 * @defaultvalue 0.7
+	 * @since 1.0.0-rc.13
+	 */
+	@property({ validator: Float, defaultValue: STEP_SWITCH_THRESHOLDS.DEFAULT })
+	stepSwitchThreshold!: number;
+
+	/**
+	 * Defines the height of the <code>ui5-wizard</code> content.
+	 * @private
+	 */
+	@property({ validator: Float })
+	contentHeight!: number;
+
+	@property({ multiple: true })
+	_groupedTabs!: Array<WizardTab>
+
+	@property()
+	_breakpoint!: string
+
+	/**
+	 * Defines the steps.
+	 * <br><br>
+	 * <b>Note:</b> Use the available <code>ui5-wizard-step</code> component.
+	 *
+	 * @type {sap.ui.webc.fiori.IWizardStep[]}
+	 * @name sap.ui.webc.fiori.Wizard.steps
```

Comment:
Properties are added to prototype of the class. Also default slot is added as default property of the prototype (to appear in the documentation as default)
```
sap.ui.webc.fiori.Wizard.prototype.default
```